packages/twenty-server/src/workspace/workspace-query-runner/workspace-query-runner.module.ts

```diff
@@ -13,7 +14,11 @@ import { WorkspaceQueryRunnerService } from './workspace-query-runner.service';
     WorkspaceDataSourceModule,
     WorkspacePreQueryHookModule,
   ],
-  providers: [WorkspaceQueryRunnerService, ...workspaceQueryRunnerFactories],
+  providers: [
+    WorkspaceQueryRunnerService,
+    ...workspaceQueryRunnerFactories,
+    RecordPositionListener,
```

Comment:
@charlesBochet @magrinj I don't know if the service / job / factory / listener should live into the query runner folder. Is there a better place ?